DEVELOPMENT.md

```diff
@@ -91,15 +112,16 @@ accessible from your development cluster. So you can run the receive adapter
 kubernetes cluster to use for the ConfigMap-based bookmarking
 ([issue](https://github.com/vmware-tanzu-private/sources-for-knative/issues/16)).
 
-Store the credentials on the filesystem:
+Store the credentials on the filesystem in a writable path:
```

Comment:
Might want to mention why this needs to be writable?

Reply:
Sorry, wrong wording. Reason I mention this is that previously, at least on OSX, the fixed `const MountPath` was in a non-writable location. Will simply remove this wording.